Sources/Orbit/Components/InputField.swift

```diff
@@ -25,37 +27,79 @@ public struct InputField: View {
     let keyboard: UIKeyboardType
     let autocapitalization: UITextAutocapitalizationType
     let isAutocompleteEnabled: Bool
+    let isSecure: Bool
     let message: MessageType
-
     let onEditingChanged: (Bool) -> Void
     let onCommit: () -> Void
     let suffixAction: () -> Void
 
     public var body: some View {
         VStack(alignment: .leading, spacing: 1) {
             FormFieldLabel(label)
-            
+
             InputContent(
                 prefix: prefix,
                 suffix: suffix,
                 state: state,
                 message: message,
                 isEditing: isEditing,
-                suffixAction: suffixAction
-            ) {
-                HStack(spacing: 0) {
-                    textField
-                    Spacer(minLength: 0)
-                    clearButton
+                suffixAction: suffixAction,
+                label: {
+                    HStack(spacing: 0) {
+                        input
+                            .textFieldStyle(TextFieldStyle(leadingPadding: 0))
+                            .autocapitalization(autocapitalization)
+                            .disableAutocorrection(isAutocompleteEnabled == false)
+                            .textContentType(textContent)
+                            .keyboardType(keyboard)
+                            .font(.orbit(size: Text.Size.normal.value, weight: .regular))
+                            .accentColor(.blueNormal)
+                            .frame(height: Layout.preferredButtonHeight)
+                            .background(textFieldPlaceholder, alignment: .leading)
+                            .disabled(state == .disabled)
```

Comment:
Secure version does not respect disabled color style (looks as if it is active).